dashboard/src/containers/AppUpgradeContainer/AppUpgradeContainer.test.tsx

```diff
@@ -1,15 +1,18 @@
 import { shallow } from "enzyme";
+import { Location } from "history";
 import configureMockStore from "redux-mock-store";
 import thunk from "redux-thunk";
 import AppUpgrade from "../../components/AppUpgrade";
 import Upgrade from "./AppUpgradeContainer";
 
 const mockStore = configureMockStore([thunk]);
 
-const emptyLocation = {
+const emptyLocation: Location = {
```

Comment:
Just adding the type here (as in other files) and adding the required fields we were missing.